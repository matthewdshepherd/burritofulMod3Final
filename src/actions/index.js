export const setOrders = orders => ({
  type: 'SET_ORDERS',
  orders
});

export const addOrder = (allOrders, order) => ({
  type: 'ADD_ORDER',
  allOrders,
  order
});
