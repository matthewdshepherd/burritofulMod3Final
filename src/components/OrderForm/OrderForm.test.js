import React from 'react'
import { shallow } from 'enzyme'
import { OrderForm, mapStateToProps, mapDispatchToProps } from './OrderForm'
import { addOrder } from '../../actions/index'

jest.mock('../../apiCalls.js')

describe('OrderForm', () => {
    let wrapper

    beforeEach(() => {
        wrapper = shallow(<OrderForm />)
    })

    it.skip('should match the snapshot', () => {
        expect(wrapper).toMatchSnapshot()
    })

    it('Should be able to update state when handleNameChange() is called', () => {
        const mockEvent = {
            target: {
              name: 'name',
              value: 'Wonder Woman'
            }
          }
          wrapper.instance().this.setState({id: 1})
          wrapper.instance().handleNameChange(mockEvent);

          expect(wrapper.state('name')).toEqual(mockEvent.target.value);

    })

    // it('Should be able to update state when handleIngredientChange() is called', () => {
    //     const mockEvent = {
    //         target: {
    //           name: 'name',
    //           value: 'Wonder Woman'
    //         }
    //       }

    //       wrapper.instance().handleNameChange(mockEvent);

    //       expect(wrapper.state('name')).toEqual(mockEvent.target.value);

    // })
})