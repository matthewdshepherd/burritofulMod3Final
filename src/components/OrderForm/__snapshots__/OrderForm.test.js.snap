// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OrderForm should match the snapshot 1`] = `undefined`;
